components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/PolicyDAOImpl.java

```diff
@@ -1153,4 +1285,107 @@ public void deleteSubscriptionPolicy(String policyName) throws APIMgtDAOExceptio
     }
 
 
+    private void deleteAPIPolicyByUuid(String uuid) throws APIMgtDAOException {
+        String sqlQuery = "DELETE FROM AM_API_POLICY WHERE UUID = ?";
+
+        try (Connection connection = DAOUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(sqlQuery))    {
+            preparedStatement.setString(1, uuid);
+            preparedStatement.execute();
+        } catch (SQLException e)    {
+            log.error("An Error occurred while deleting Policy with uuid [" + uuid + "], " , e);
+            throw new APIMgtDAOException("Error occurred while deleting Policy with uuid : " + uuid, e);
```

Comment:
Change the error message to same format and add it to string and use it in both log and throw.

String msg = An Error occurred while deleting Policy with uuid [" + uuid + "] "
log.error(msg,e);
throw new APIMgtDAOException(msg,e);